setup_np4intel_dev_env.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
```

Comment:
I think this could be merged into `setup_dev_env.sh` with a few variables. No need to have 90% duplication.

Reply:
probably true, but I figured given that they both build on different docker container bases (i.e. stratum uses debian, stratum for np4intel uses Ubuntu) then it might make sense to have two different scripts, but I guess we could have an option instead.  You'll note there is also a different builder container that needs to be created... again because it's based on Ubuntu and also installs the NP4 SDK libraries.

Reply:
I've merged them into the same setup_dev_env.sh script, they use different caches, docker files and image names.